Blueprints/BlueprintDefinitions/vs2022/EmptyServerless/template/src/BlueprintBaseName.1/Startup.cs

```diff
@@ -0,0 +1,36 @@
+using Amazon.Lambda.Annotations;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace BlueprintBaseName._1
```

Comment:
nit: could we stick to using top level statements?

Reply:
Done